README.md

```diff
@@ -78,16 +68,15 @@ npm run test
 
 ### Build
 
-Before you build your lens, you should update your lens metadata in `./SuperDuperLens/lens.json`.
-Note: "lensEventApiVersion" currently defaults to 1.
+Before you build your lens, you should update your lens project data in `./lens-name/package.json`.
 
-Package up all your lens resources into `./dist/SuperDuperLens.zip` for deployment using the build script:
+Package up all your lens resources into `lens-name.zip` for deployment using the build script:
 
 ```
 npm run build
 ```
 
-During the build process, webpack bundles up all the javascript files and dependencies required by `./SuperDuperLens/src/main.js` into `./SuperDuperLens/lens.js`, then that gets zipped up into `./dist/SuperDuperLens.zip` along with `./SuperDuperLens/lens.json`.
+During the build process, webpack bundles up all the javascript files and dependencies required by `./lens-name/src/main.js` into `./lens-name/lens.js`, then that gets zipped up into `lens-name.zip` in your lens project along with `./lens-name/package.json`.
```

Comment:
you get the gist of all these comments, right? try to *not* use `lens-name` anywhere?

Reply:
I'll update all the lens-name sentences